### DISCUSSION TITLE
Tunny v1.0.0 beta 1

### DISCUSSION THREAD

**Discussion by hrntsm** [Tunny-gh/Tunny_in_Open_Sea#357] [Announcements]
## Summary

Tunny v1.0 beta1.
It is now the latest optimised component with a completely new UI. However, as it is a completely new UI, it is still considered buggy. Therefore it is released as beta.

Your feedback will make Tunny v1 even better.
If you find any bugs, please post them in Discussion
Thank you for your cooperation.

![image](https://github.com/user-attachments/assets/8306e938-ea2e-49cd-97ed-14ac4eb6405b)

## What's Changed

* Fix/settings deserialize exception by @hrntsm in https://github.com/hrntsm/Tunny/pull/313
* Feature/update log output setting by @hrntsm in https://github.com/hrntsm/Tunny/pull/314
* Feature/add tutorial menu by @hrntsm in https://github.com/hrntsm/Tunny/pull/315
* Feature/disable viewport drawing by @hrntsm in https://github.com/hrntsm/Tunny/pull/316
* Fix/fish print by path lock image file by @hrntsm in https://github.com/hrntsm/Tunny/pull/317
* Add citation by @hrntsm in https://github.com/hrntsm/Tunny/pull/321
* Update CITATION.cff by @hrntsm in https://github.com/hrntsm/Tunny/pull/322
* Fix TSettings deserialization in LoadingInstruction.cs by @hrntsm in https://github.com/hrntsm/Tunny/pull/323
* Refactor logging configuration to stop outputting logs to console by @hrntsm in https://github.com/hrntsm/Tunny/pull/324
* Fix disable window minimize button by @hrntsm in https://github.com/hrntsm/Tunny/pull/325
* Fix _stop_flag exception by @hrntsm in https://github.com/hrntsm/Tunny/pull/328
* Clean Human in the loop code  by @hrntsm in https://github.com/hrntsm/Tunny/pull/329
* Add Pruner by @hrntsm in https://github.com/hrntsm/Tunny/pull/331
* Fix/3d pareto front by @hrntsm in https://github.com/hrntsm/Tunny/pull/332
* Fix condition for setting user attribute in OptimizeLoop.cs by @hrntsm in https://github.com/hrntsm/Tunny/pull/333
* Featura/wpf UI by @hrntsm in https://github.com/hrntsm/Tunny/pull/335
* Clean message box by @hrntsm in https://github.com/hrntsm/Tunny/pull/336
* Fix/percentile pruner by @hrntsm in https://github.com/hrntsm/Tunny/pull/337
* Feature/add objective & variable parameter settings by @hrntsm in https://github.com/hrntsm/Tunny/pull/338
* Feature/update python lib by @hrntsm in https://github.com/hrntsm/Tunny/pull/339
* Feature/support some optunahub sampler by @hrntsm in https://github.com/hrntsm/Tunny/pull/340
* Fix pruner `auto` value parse by @hrntsm in https://github.com/hrntsm/Tunny/pull/341
* Feature/add icon by @hrntsm in https://github.com/hrntsm/Tunny/pull/342
* Introduce task complete source to wait solution end by @hrntsm in https://github.com/hrntsm/Tunny/pull/343
* Add minimize Rhino window function by @hrntsm in https://github.com/hrntsm/Tunny/pull/344
* Feature/input validater by @hrntsm in https://github.com/hrntsm/Tunny/pull/345
* Enable default optimization settings button by @hrntsm in https://github.com/hrntsm/Tunny/pull/346
* Add tell specified value when pruned by @hrntsm in https://github.com/hrntsm/Tunny/pull/348
* Update PythonInstaller to include new ui by @hrntsm in https://github.com/hrntsm/Tunny/pull/349
* Add study group interaction by @hrntsm in https://github.com/hrntsm/Tunny/pull/351
* Output user_attrs to TT-DesignExplorer by @hrntsm in https://github.com/hrntsm/Tunny/pull/352
* Fix/example dir load error by @hrntsm in https://github.com/hrntsm/Tunny/pull/354
* Feature/output by @hrntsm in https://github.com/hrntsm/Tunny/pull/355
* Delete WinForm UI by @hrntsm in https://github.com/hrntsm/Tunny/pull/356

**Full Changelog**: https://github.com/hrntsm/Tunny/compare/v0.12.0...v1.0.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/hrntsm/Tunny/releases/tag/v1.0.0-beta.1'>Tunny v1.0.0 beta 1</a>.</em>

**Comment by SpaceArchitecture** [comment]
Looks good, hopefully this update will iron out the previous bugs

**Comment by ChenChihYuan** [comment]
I might have found one bug. I am using `0.12.0` version of Tunny. But sometimes it failed to show the website after I press the `TT-DesignExplorer`. I will check if it happens in 1.0.0 version.
![image](https://github.com/user-attachments/assets/3d94a137-c610-4d4d-a1a8-17086b0d3901)

Thanks for the great work

[Edited]
Now that I can see the DesignExplorer. I think maybe just it won't automatic pop up the page. But I can still visit http://127.0.0.1:8081/index.html to get the result

**Reply by hrntsm** [reply]
@ChenChihYuan 

Thank you your feed back.
I had not thought about using both Dashboard and TT-DesignExplorer at the same time, so the specifications are a little inconvenient to operate.

I are currently working on a development version that will allow you to check images and parameters on the Dashboard like in DesignExplorer, as shown in the image below.

![image](https://github.com/user-attachments/assets/7fc98196-8fcc-49ab-9e89-7c4581d29672)

**Comment by SpaceArchitecture** [comment]
How are things going ? Did you get enough funding to push v1.0 ?

**Reply by hrntsm** [reply]
I have not been able to raise any funds at all.
So I am considering a license subscription.

**Comment by SpaceArchitecture** [comment]
Sure, that's an option


Thomas Lagarde, Associate





T 202.628.1033

M 608.886.1862



***@***.***

www.ayerssaintgross.com<http://www.ayerssaintgross.com/>

________________________________
